.github/workflows/integration-test.yml

```diff
@@ -29,11 +29,12 @@ jobs:
 
       - name: Install go-junit-report
         run: go install github.com/jstemmer/go-junit-report@latest
-
+      
```

Comment:
This whitespace change isn't needed

Reply:
Looks like my IDE went rogue and put a tab in there. Removed now.